homeworks/vlad.makedon_Vladmaked/homework_3/recursion.js

```diff
@@ -0,0 +1,18 @@
+const rl = require('readline').createInterface(process.stdin, process.stdout);
+
+rl.question('Enter the beginning of the range: ', (startRange) => {
+    rl.question('Enter end of range: ', (endRange) => {
+        rl.close();
+        function getNum(n, start, end) {
```

Comment:
the idea of this function is to take one argument, not three
separate the task logic into two parts:

1. recursive function
2. loop to utilize this function

Reply:
Done